src/features/contactmoment/ContactmomentVragenMenu.vue

```diff
@@ -4,6 +4,14 @@
     message="Let op, je hebt het contactverzoek niet afgerond. Als je deze vraag verlaat, wordt het contactverzoek niet verstuurd."
   />
   <menu class="vragen-menu">
+    <li>
+      <button
+        class="icon-after plus new-question"
+        type="button"
+        title="Nieuwe vraag"
+        @click="startNieuweVraag"
+      ></button>
+    </li>
     <li v-for="(vraag, idx) in contactmomentStore.vragen" :key="idx">
       <span v-if="vraag === contactmomentStore.huidigeVraag">
         {{ idx + 1 }}
```

Comment:
We're checking if the `vraag === huidigeVraag` and then creating either a `span` or a `button`. However, I think both cases it could be a button, only in the active state the button would become disabled, like so:

```
<li v-for="(vraag, idx) in contactmomentStore.vragen" :key="idx">
  <button
    type="button"
    :title="`Ga naar vraag ${idx + 1}`"
    :disabled="vraag === contactmomentStore.huidigeVraag"
    :class="{ current: vraag === contactmomentStore.huidigeVraag }" // Also setting a current state here, see other comments
    @click="switchVraag(vraag)"
  >
    {{ idx + 1 }}
  </button>
</li>
```